### DISCUSSION TITLE
Beta 版 RemNote 已解决搜索时输入中文崩溃问题

### DISCUSSION THREAD

**Discussion by lovedebug** [5eagull/RemNote-Chinese-chat-group#21] [知识分享]
付费用户可以考虑加入体验 https://beta.remnote.io/beta_server

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/69051504/115731932-1c5c7d00-a3ba-11eb-8883-cb7c56360ef4.png)
太棒了，我测试了下的确不会崩溃了，版本号为1.3.7V

**Reply by makerjackie** [reply]
不过 PC 版一直不会崩溃
